cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -1507,11 +1511,39 @@ protected String getServiceAccountName() {
         return varList;
     }
 
+    /**
+     * Validates provided OAuth configuration. Throws InvalidResourceException when OAuth configuration contains forbidden combinations.
+     *
+     * @param oAuth     OAuth type authentication object
+     */
+    private static void validateOauth(KafkaListenerAuthenticationOAuth oAuth) {
+        if (oAuth.getValidIssuerUri() == null) {
+            log.error("Valid Issue URI has to be specified");
+            throw new InvalidResourceException("Valid Issue URI has to be specified");
+        }
+
+        if (oAuth.getJwksExpirySeconds() != null && oAuth.getJwksRefreshSeconds() != null && oAuth.getJwksExpirySeconds() < oAuth.getJwksRefreshSeconds() + 60) {
```

Comment:
I think that in case only one of these is set, you still should check it against the defaults.